app/helpers/post_location_helper.rb

```diff
@@ -0,0 +1,9 @@
+module PostLocationHelper
+  def link_to_post(post)
+		if post.type == 'ExternalPost'
+			link_to(post.title, post.source_url, target: '_blank') 
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.<br>Trailing whitespace detected.